tests/contrib/actions/reports/test_files.py

```diff
@@ -0,0 +1,43 @@
+try:
+    from unittest.mock import MagicMock, patch, mock_open
+except ImportError:
+    from mock import MagicMock, patch, mock_open
+import pytest
+from unittest import TestCase
+
+from spidermon.contrib.actions.reports.files import CreateFileReport
+from spidermon.exceptions import NotConfigured
+
+TEMPLATE = "template"
+FILENAME = "filename"
+
+
+class CreateFileReportTests(TestCase):
```

Comment:
`pytest` doesn't require a `TestCase` class for the tests. All tests in Spidermon are single function.